lib/matplotlib/widgets.py

```diff
@@ -514,53 +516,48 @@ class CheckButtons(AxesWidget):
     """
     A GUI neutral set of check buttons.
 
-    For the check buttons to remain responsive you must keep a
-    reference to this object.
+    For the check buttons to remain responsive you must keep a reference to
+    this object.
 
-    The following attributes are exposed
+    Connect to the CheckButtons with the :meth:`on_clicked` method.
 
-     *ax*
-        The :class:`matplotlib.axes.Axes` instance the buttons are
-        located in
+    Attributes
+    ----------
+     ax
```

Comment:
Please make really sure you adhere to the documentation conventions. That includes indentation in docstrings, on which ReST/sphinx/numpydoc is very strict.

I know it's cumbersome, but we need a certain quality for PRs to be merged; and I don't have the time to comment on every formatting issue.

At best, build the docs locally and check the result. Alternatively, check the documentation built via CircleCI
![grafik](https://user-images.githubusercontent.com/2836374/52821522-51206b80-30af-11e9-94f0-8e419bac972b.png)
![grafik](https://user-images.githubusercontent.com/2836374/52821549-64cbd200-30af-11e9-8dd8-d5e39990b4a2.png)

You'll see that the formatting is messed because the indentations here are all not right.

https://18374-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/api/widgets_api.html?highlight=widgets#matplotlib.widgets.CheckButtons

Also, if you convert a docstring from the old format to numpydoc, I expect the conversion to be complete, i.e. add types, no parameter formatting etc.